docs/lms_config.ini

```diff
@@ -32,10 +31,17 @@ progress = SFD_check_progress.html
 
 learneraccountsettings = sfd_dashboard_profile/SFD_dashboard_settings_profile.html#exploring-the-account-settings-page
 
-learnerdashboard = sfd_dashboard_profile/SFD_dashboard_settings_profile.html#exploring-the-learner-dashboard
+learnerdashboard = sfd_dashboard_profile/SFD_dashboard_settings_profile.html#exploring-the-dashboard
+
+programs = SFD_enrolling.html#programs
 
 bookmarks = SFD_bookmarks.html
 
```

Comment:
They look that way in the pull request, but in Sublime all of the values are flush. Maybe it's the presence/absence of a + that makes them look that way?